src/main/java/ppp/discord/bot/model/User.java

```diff
@@ -1,4 +1,67 @@
 package ppp.discord.bot.model;
 
+import discord4j.common.util.Snowflake;
+import discord4j.core.event.domain.message.MessageCreateEvent;
+import java.util.Objects;
+
 public class User {
+
+    private Snowflake id;
```

Comment:
Why are we using this "Snowflake" object?

Reply:
It's the ID of the Member of discord